admin_guide/upgrading_citus.rst

```diff
@@ -7,67 +7,242 @@ $$$$$$$$$$$$$$$
 
 Upgrading Citus Versions
 ########################
-Upgrading the Citus version requires first obtaining the new Citus extension and
-then installing it in each of your database instances. The first step varies by
-operating system.
 
-.. _upgrading_citus_package:
+Citus adheres to `semantic versioning <http://semver.org/>`_ with patch-, minor-, and major-versions. The upgrade process differs for each, requiring more effort for bigger version jumps.
 
-Step 1. Update Citus Package
-----------------------------
-
-**OS X**
+Upgrading the Citus version requires first obtaining the new Citus extension and then installing it in each of your database instances. Citus uses separate packages for each minor version to ensure that running a default package upgrade will provide bug fixes but never break anything. Let's start by examining patch upgrades, the easiest kind.
 
-::
+Patch Version Upgrade
+---------------------
 
-  brew update
-  brew upgrade citus
+To upgrade a Citus version to its latest patch, issue a standard upgrade command for your package manager. Assuming version 6.1 is currently installed:
 
 **Ubuntu or Debian**
 
-::
+.. code-block:: bash
 
   sudo apt-get update
```

Comment:
`apt-get update` ≠ `yum update`. Update means "update the package system metadata", so it's necessary to make sure your system even knows there is a new Citus available.